packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.scss

```diff
@@ -0,0 +1,88 @@
+@use "sass:math";
+
+@import "../../style/index.scss";
+
+.create-service-wizard {
+  display: flex;
+  height: 100%;
+  flex-direction: column;
+
+  &__splitWrapper {
+    display: flex;
+    height: 100%;
+  }
+
+  &__left {
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background: #22273c;
```

Comment:
I see that we hardcode a lot of HEX values for colors. Do we not have a central place where we refer to our design system colors?

Reply:
@arielweinberger You right, i will update it